vowpalwabbit/OjaNewton.cc

```diff
@@ -529,24 +529,30 @@ void save_load(OjaNewton& ON, io_buf& model_file, bool read, bool text)
   }
 }
 
-base_learner* OjaNewton_setup(arguments& arg)
+base_learner* OjaNewton_setup(VW::config::options_i& options, vw& all)
 {
   auto ON = scoped_calloc_or_throw<OjaNewton>();
-  if (arg.new_options("OjaNewton options")
-      .critical("OjaNewton", "Online Newton with Oja's Sketch")
-      ("sketch_size", ON->m, 10, "size of sketch")
-      ("epoch_size", ON->epoch_size, 1, "size of epoch")
-      ("alpha", ON->alpha, 1.f, "mutiplicative constant for indentiy")
-      ("alpha_inverse", po::value<float>(), "one over alpha, similar to learning rate")
-      ("learning_rate_cnt", ON->learning_rate_cnt, 2.f, "constant for the learning rate 1/t")
-      ("normalize", ON->normalize, true, "normalize the features or not")
-      ("random_init", ON->random_init, true, "randomize initialization of Oja or not").missing())
+
+  bool oja_newton;
+  float alpha_inverse;
+  VW::config::option_group_definition new_options("OjaNewton options");
+  new_options.add(VW::config::make_typed_option("OjaNewton", oja_newton).keep().help("Online Newton with Oja's Sketch"));
```

Comment:
This is adding quite a bit of verbage compared to what was there before.  

Why can't we use an operator() similar to previous?  Or at least .add() if you want to explicit?

And, the repeated use of VW::config:: is kind of unfortunate.

Reply:
using the namespace reduces noise here by quite a bit, I'd be happy to do that to all of them. Using operator() is possible here, I just avoided it because clang-format has some trouble formatting it nicely and really just personal style preferences.

What do you mean by?

> Or at least .add() if you want to explicit?